ercc/registry/registry_test.go

```diff
@@ -0,0 +1,339 @@
+package registry_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/attestation"
+	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/registry"
+	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/registry/mocks"
+	"github.com/hyperledger-labs/fabric-private-chaincode/internal/protos"
+	"github.com/hyperledger-labs/fabric-private-chaincode/internal/utils"
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+	"github.com/hyperledger/fabric-protos-go/ledger/queryresult"
+	"github.com/hyperledger/fabric-protos-go/peer/lifecycle"
+	"github.com/hyperledger/fabric/protoutil"
+	"github.com/stretchr/testify/require"
+)
+
+//go:generate counterfeiter -o mocks/transaction.go -fake-name TransactionContext . transactionContext
```

Comment:
was there a strong reason change the outpudle directory and fake-name?  I actually like the default name and distinguishes from the normal mocks (see my other comment that i got confused by the name "mock" because this mock doesn't match the other mocks which are usually self-contained functional replacements for a module rather than the kindof  "remote-controlled" ones here.   That said, no strong reason but it confused me and using defaults even makes these comments more consice?

Reply:
first reason, I got inspired by some package in fabric using counterfeiter with this naming convention. And by looking at the output of the default setting, I like this non-standard format more. I think we maybe rename the `mocks` folder to `fakes` to do something against you mental picture of mocks and "remote-controlled" fakes. :)

CHanging also the interface names to default with FakeEverything naming I don't like so much as the fake package already notes that this is a "mock" or "fake" implementation.